src/react/ui-elements/TruncatedText.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import React from 'react';
```

Comment:
If we don't test this component now, we will have to do it after the release.

Reply:
done